src/rendererbase.h

```diff
@@ -120,9 +120,9 @@ class RendererBase : public apf::MimoProcessor<Derived
         , reference_offset_orientation(fifo)
         , master_volume(fifo, 1)
         , processing(fifo, true)
-        , decay_exponent(fifo, params.get("decay_exponent", 1))
+        , decay_exponent(fifo, params.get<sample_type>("decay_exponent", 1))
```

Comment:
Should this use 1.0f instead of 1?

Reply:
We had that discussion in src/rendererbase.h above.
It doesn't matter, and @mgeier preferred it this way.
Since I already squashed, this is a lot harder to see now...

Reply:
Long story short:
Here, we explicitly specify the template parameter to `sample_type`.

Reply:
For reference, this is the discussion from above: https://github.com/SoundScapeRenderer/ssr/pull/88#discussion_r168017444

It turns out that `apf::parameter_map` is unsafe and we should fix it some day. I just created #104 as a reminder.